### PR TITLE
Add RSS

### PR DESCRIPTION
**Pull request by ysm-dev**
- https://www.rssboard.org/rss-specification
- https://en.wikipedia.org/wiki/RSS

examples
- https://latinometrics.substack.com/
- https://dev.to/
- https://medium.com/@robertroybritt
- https://nextjs.org/
- https://addyosmani.com/

### CODE REVIEW

File: src/technologies/r.json [hunk 1]
```diff
@@ -1895,6 +1895,18 @@
     ],
     "website": "https://www.royalmail.com"
   },
+  "RSS": {
+    "cats": [
+      19
```
Comment (ysm-dev) [inline src/technologies/r.json:1900]: I'm not sure this is the right category,
I considered between 49(feed readers), 19(Miscellaneous).
But I think it's not "readers", so I set it to 19.

File: src/technologies/r.json [hunk 2]
```diff
@@ -1895,6 +1895,18 @@
...
+    ],
+    "description": "RSS is a family of web feed formats used to publish frequently updated works—such as blog entries, news headlines, audio, and video—in a standardized format.",
```
Comment (ysm-dev) [inline src/technologies/r.json:1902]: The description was generated by Copilot.

File: src/technologies/r.json [hunk 3]
```diff
@@ -1895,6 +1895,18 @@
...
+    "dom": [
+      "link[type=\"application/rss+xml\"]",
+      "link[type=\"application/atom+xml\"]"
+    ],
+    "icon": "RSS.png",
```
Comment (nurbek91) [inline src/technologies/r.json:1907]: RSS
32x32 svg

File: src/technologies/r.json [hunk 4]
```diff
@@ -1895,6 +1895,18 @@
     ],
     "website": "https://www.royalmail.com"
   },
+  "RSS": {
+    "cats": [
+      19
+    ],
+    "description": "RSS is a family of web feed formats used to publish frequently updated works—such as blog entries, news headlines, audio, and video—in a standardized format.",
+    "dom": [
```
Comment (nurbek91) [inline src/technologies/r.json:1903]: Maybe try testing like this, for me this option seems more flexible and readable. Maybe your version is also working, but I did not test it

```
 "dom": {
      "link[type*='application']": {
        "attributes": {
          "type": "application/(?:rss|atom)+xml"
        }
      }
    },
```

**Review by nurbek91 — COMMENTED** [review]
Hey mate, thanks for PR!!

File: src/technologies/r.json (hunk 3 above)
Comment (ysm-dev) [reply src/technologies/r.json]: [7814219](https://github.com/wappalyzer/wappalyzer/pull/6945/commits/7814219c0ca32a76dbd03d96f240dc38fd326ad8)

File: src/technologies/r.json (hunk 4 above)
Comment (ysm-dev) [reply src/technologies/r.json:1903]: I tested your code with cli
```zsh
node src/drivers/npm/cli.js https://addyosmani.com/ --pretty
```

but it doesn't detect RSS. 🤔

Comment (AliasIO) [reply src/technologies/r.json]: The `+` needs to be escaped.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**